src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -616,6 +621,34 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         }
     }
 
+    winrt::fire_and_forget TermControl::_RendererWarning(const HRESULT hr)
+    {
+        auto weakThis{ get_weak() };
+        co_await winrt::resume_foreground(Dispatcher());
+
+        if (auto control{ weakThis.get() })
+        {
+            winrt::hstring message;
+
+            if (E_FILE_NOT_FOUND == hr)
+            {
+                message = { fmt::format(std::wstring_view{ RS_(L"PixelShaderNotFound") }, _settings.PixelShaderPath()) };
+            }
+            else if (D2DERR_SHADER_COMPILE_FAILED == hr)
+            {
+                message = { fmt::format(std::wstring_view{ RS_(L"PixelShaderCompileFailed") }) };
+            }
+            else
+            {
+                message = { fmt::format(std::wstring_view{ RS_(L"UnexpectedRendererError") }, hr) };
+                // message{ fmt::format(L"Renderer encountered an unexpected error: {}", hr) };

```

Comment:
Leftovers?

Reply:
I just hate forgetting to push before submitting the PR
![image](https://user-images.githubusercontent.com/18356694/102081843-821a7600-3dd6-11eb-91ff-c78a9ac74f06.png)